tests/integration/cli/tests/run.rs

```diff
@@ -546,3 +546,53 @@ fn run_no_start_wasm_report_error() -> anyhow::Result<()> {
     assert_eq!(result.contains("Can not find any export functions."), true);
     Ok(())
 }
+
+// Test that changes to wapm run don't break wasmer run
+#[test]
+fn test_wapm_run_works() -> anyhow::Result<()> {
```

Comment:
We should test wasmer run with the complex url (in windows) rather than depend on wapm

Reply:
done, only CI has to pass now